web/src/app/core/services/project.service.ts

```diff
@@ -70,7 +85,10 @@ export class ProjectService {
       );
   }
 
-  // This function delete the project via uid
+  /**
+   * Delete the project using project uid
+   * @param uid uid of project to be deleted
```

Comment:
In the compodocs example they include the type..

```typescript
/**
 * @example
 * This is a good example
 * processTarget('yo')
 *
 * @param {string} target  The target to process see {@link Todo}
 * @returns The processed target number
 */
function processTarget(target:string):number;
```

Reply:
But it looks like redundant to me to add the type in comment .